types/types.go

```diff
@@ -64,6 +64,38 @@ func Nullable(typ interface{}) sql.Scanner {
 		return &nullDuration{typ}
 	case sql.Scanner:
 		return &nullable{typ}
+	case **string:
+		return &nullPtrString{typ}
+	case **bool:
+		return &nullPtrBool{typ}
+	case **int8:
+		return &nullPtrInt8{typ}
+	case **uint8:
+		return &nullPtrUint8{typ}
+	case **int16:
+		return &nullPtrInt16{typ}
+	case **uint16:
+		return &nullPtrUint16{typ}
+	case **uint:
+		return &nullPtrUint{typ}
+	case **int:
+		return &nullPtrInt{typ}
+	case **uint32:
+		return &nullPtrUint32{typ}
+	case **int32:
+		return &nullPtrInt32{typ}
+	case **uint64:
+		return &nullPtrUint64{typ}
+	case **int64:
+		return &nullPtrInt64{typ}
+	case **float32:
+		return &nullPtrFloat32{typ}
+	case **float64:
+		return &nullPtrFloat64{typ}
+	case **time.Time:
+		return &nullPtrTime{typ}
+	case **time.Duration:
+		return &nullPtrDuration{typ}
 	}
 
 	return &nullableErr{typ}
```

Comment:
if a type is not on the switch, it's a `*nullableErr`, which is just a type that will return an error either using `Value` or `Scan`